cloudinary-image-management-and-manipulation-in-the-cloud-cdn/php/class-settings-page.php

```diff
@@ -290,6 +296,14 @@ public function render_field( $field, $value = null, $show_description = true )
 				<input type="<?php echo esc_attr( $type ); ?>" class="cld-field regular-<?php echo esc_attr( $type ); ?>" id="<?php echo esc_attr( $field['label_for'] ); ?>" name="<?php echo esc_attr( $setting_slug ); ?>[<?php echo esc_attr( $field['slug'] ); ?>]" <?php if ( ! empty( $field['pattern'] ) ) : ?>pattern="<?php echo esc_attr( $field['pattern'] ); ?>"<?php endif; ?> data-condition="<?php echo esc_attr( $condition ); ?>" data-context="<?php echo esc_attr( $context ); ?>" <?php echo esc_attr( $required ); ?> <?php checked( 'on', $value ); ?>>
 				<?php
 				break;
+			case 'radio':
+				foreach ( $field['choices'] as $key => $option ) :
+				?>
+					<input type="<?php echo esc_attr( $type ); ?>" class="cld-field regular-<?php echo esc_attr( $type ); ?>" id="<?php echo esc_attr( $field['label_for'] . '_' . $key ); ?>" name="<?php echo esc_attr( $setting_slug ); ?>[<?php echo esc_attr( $field['slug'] ); ?>]" <?php if ( ! empty( $field['pattern'] ) ) : ?>pattern="<?php echo esc_attr( $field['pattern'] ); ?>"<?php endif; ?> data-condition="<?php echo esc_attr( $condition ); ?>" data-context="<?php echo esc_attr( $context ); ?>" <?php echo esc_attr( $required ); ?> <?php checked( $key, $value ); ?> value="<?php echo esc_attr( $key ); ?>"> 
+					<label for="<?php echo esc_attr( $field['label_for'] . '_' . $key ); ?>"><?php echo $option; ?></label>
```

Comment:
@dugajean 

It seems that we aren't escaping `$option`.

Reply:
Good catch 👍